cli/src/commands/gerrit/upload.rs

```diff
@@ -88,6 +88,123 @@ pub struct UploadArgs {
     /// Do not actually push the changes to Gerrit
     #[arg(long = "dry-run", short = 'n')]
     dry_run: bool,
+
+    // The following flags are options gerrit supports during upload.
+    // They are documented at
+    // https://gerrit-review.googlesource.com/Documentation/user-upload.html
+    /// Add these emails as a reviewer.
+    #[arg(long)]
+    reviewer: Vec<String>,
+
+    /// CC these emails on the change.
+    #[arg(long)]
+    cc: Vec<String>,
+
+    /// Add the following label configured by gerrit.
+    /// Defaults to +1 if no value is set.
+    /// Eg. --label=Auto-Submit will set the Auto-Submit label (if it exists
+    /// on your gerrit host)
+    #[arg(long, short = 'l')]
+    label: Vec<String>,
+
+    /// Applies a topic to the change.
+    /// See https://gerrit-review.googlesource.com/Documentation/intro-user.html#topics.
+    /// Changes can be grouped by topic, and gerrit can be configured to submit
+    /// all changes in a topic together in a single click.
+    #[arg(long)]
+    topic: Option<String>,
+
+    /// Applies a hashtag to a change.
+    /// See https://gerrit-review.googlesource.com/Documentation/intro-user.html#hashtags.
+    /// Hashtags are freeform strings associated with a change, like on social
+    /// media platforms. Similar to topics, hashtags can be used to group
+    /// related changes together, and to search using the hashtag: operator.
+    /// Unlike topics, a change can have multiple hashtags, and they are only
+    /// used for informational grouping. Changes with the same hashtags are
+    /// not necessarily submitted together.
+    #[arg(long)]
+    hashtag: Option<String>,
+
+    /// Marks a change as WIP (work in progress).
+    /// See https://gerrit-review.googlesource.com/Documentation/intro-user.html#wip.
+    #[arg(long)]
+    wip: bool,
+    /// Unmarks a change as WIP (work in progress).
+    #[arg(long)]
+    ready: bool,
+
+    /// Marks a change as private.
+    /// See https://gerrit-review.googlesource.com/Documentation/intro-user.html#private-changes.
+    #[arg(long)]
+    private: bool,
+    /// Unmarks a change as private.
+    #[arg(long)]
+    remove_private: bool,
+
+    /// Publishes any draft comments for the given change.
+    #[arg(long)]
+    publish_comments: bool,
+    /// Disables publishing of any draft comments for the given change.
+    /// This is only useful if the user has configured gerrit to publish
+    /// comments by default.
+    #[arg(long)]
+    no_publish_comments: bool,
+
+    /// Who to email notifications to. If not provided, defaults to all.
+    #[arg(long, value_enum)]
+    notify: Option<EmailNotification>,
+
+    /// Directly submit the changes, bypassing code review.
+    #[arg(long)]
+    submit: bool,
+    /// A justification for --submit.
+    /// Note: This option is used internally in google.
+    /// This value is ignored in the upstream version of gerrit.
+    #[arg(long)]
+    justification: Option<String>,
```

Comment:
I'm not a huge fan of this in particular.

Reply:
Sure, happy to remove this. I only added it in the first place because it was shown in the official gerrit documentation.
https://gerrit-review.googlesource.com/Documentation/user-upload.html